Mathlib/GroupTheory/Index.lean

```diff
@@ -609,6 +609,14 @@ end MulAction
 
 namespace MonoidHom
 
+@[to_additive AddSubgroup.surjective_of_card_ker_le_div]
```

Comment:
I cannot see this lemma used in the previous additions. Is it?

Reply:
```suggestion
@[to_additive AddMonoidHom.surjective_of_card_ker_le_div]
```
`MonoidHom.surjective_of_card_ker_le_div` is used in `unitsMap_norm_surjective`; the additive version is not used. However there is a typo in the additive version (wrong namespace) which I'm now correcting.